addons/spreadsheet/static/src/list/list_data_source.js

```diff
@@ -35,13 +35,22 @@ export default class ListDataSource extends OdooViewsDataSource {
      */
     constructor(services, params) {
         super(services, params);
-        this.limit = params.limit;
+        this.biggestPosition = params.limit;
+        this.maxPositionFetched = 0;
```

Comment:
max vs biggest ? I would use only one word
```suggestion
        this.maxPosition = params.limit;
        this.maxPositionFetched = 0;
```